lib/workers/repository/process/lookup/index.ts

```diff
@@ -247,6 +266,32 @@ export async function lookupUpdates(
         newMajor: version.getMajor(fromVersion),
       });
     }
+    if (fromVersion && rangeStrategy === 'cap') {
```

Comment:
I think we need some fallback code so that if anyone configures `cap` for a versionScheme that doesn't support it then we just use `replace` instead. Not sure if this is where we put it or not.

Reply:
BTW I would expect this logic to be located next to `pin` logic.

Reply:
Should we set up some explicit mechanism to determine support for `cap`-ping, or it would be safe to assume true if `increment()` function is present?

Reply:
I think, it should be okay to make dumb implementations returning `null` if unsupported

Reply:
I think we can determine based on `increment()`